src/components/BButton/BButton.vue

```diff
@@ -1,18 +1,75 @@
 <template>
-  <button>{{text}}</button>
+  <button
+    :class="computedClass"
+    :disabled="disabled"
+    :type="type"
+    :height="height"
```

Comment:
Height and weight are handled by style, are they not?

Reply:
yep, fixed that, thanks